apps/teamsjs-diagnostic-app/package.json

```diff
@@ -11,7 +11,7 @@
     "@microsoft/teamsfx": "^2.3.2",
     "@microsoft/teamsfx-react": "^3.1.2",
     "@testing-library/dom": "^10.4.0",
-    "axios": "^1.7.3",
+    "axios": "^1.7.4",
```

Comment:
Should an update to `pnpm-lock.yaml` come in this same change?

Reply:
Yep, just pushed